### DISCUSSION TITLE
Raspberry Pi Alternative

### DISCUSSION THREAD

**Discussion by CestMoiRoma** [rudetrooper/Octoprint-Chituboard#30] [Q&A] [answered]
Hello, I'm interested in this plugin, but with the current shortage of Raspberry I looked for an alternative, and I found an affordable one, the Orange Pi Zero 2.

But that's my question: Will it work?

The card contains everything I need to bind the printer to it, but I don't know if the system will detect the printer. Has anyone tried with an alternative to Raspberry Pi?

**Comment by rudetrooper** [comment, marked as answer]
It might work if there is a USB OTG port on the board and you can setup the mass storage emulator using the Linux gadget subsystem. I'm not super familiar with the Orange Pi zero boards and I don't have any to test this on so it will probably take some work to get it working if the interfaces I mentioned earlier are available. Also we would need to be able to initiate a serial terminal via UART using the GPIO pins.

**Reply by CestMoiRoma** [reply]
Okay thanks for the advice, i will test if it's works when i receive mine

**Reply by RoninTurkey** [reply]
Just curious if you've been able to confirm if the Orange Pi Z2 works. Thanks.

**Reply by CestMoiRoma** [reply]
Yes the orange pi works great

**Reply by ojungleboy** [reply]
> Yes the orange pi works great

Hi! Which orange pi board did you use. Would be able to tell us if the steps to install it are the same for raspberry? I am not used to linux and I would like to know how difficult it is.

Thanks in advance

**Reply by CestMoiRoma** [reply]
Orange pi zero 2 and i don't remember the video but, on youtube you have a lot of tutorial who explain how to install it, it's more complicated than the raspberry pi way because you need to do all manually from scratch with the orange pi's debian modified os

**Reply by CestMoiRoma** [reply]
For a user like me who knows nothing to linux it's pretty simple i just followed the steps of the explaination.

**Reply by bvaerewyck** [reply]
works just fine with the Orange Pi Zero 2.  I have two up and running.

**Reply by ojungleboy** [reply]
Thank you legends! I found some tutorials on the internet. And I just ordered my orange pi zero 2. Thank you one again!!
